runtest.py

```diff
@@ -563,11 +563,19 @@ def find_e2e_tests(directory):
         # Skip folders containing a sconstest.skip file
         if 'sconstest.skip' in filenames:
             continue
+
+        p = Path(dirpath).joinpath( ".exclude_tests")
         try:
-            with open(os.path.join(dirpath, ".exclude_tests")) as f:
-                excludes = scanlist(f)
-        except EnvironmentError:
+            if sys.version_info.major == 3 and sys.version_info.minor < 6:
```

Comment:
Can you add a `# TODO remove when we drop PY 3.5" here?

Reply:
ok, done.